src/entities/CourseTypeFilter.ts

```diff
@@ -0,0 +1,6 @@
+export enum CourseTypeFilter {
+    ALL = "All",
+    LECTURE = "Lecture",
+    SEMINAR = "Seminar",
+    PG = "Project Group",
+}
```

Comment:
course types are part of the configuration now.. refactor all occurences of this to use the configuration in the store